backend/internal/server/apis/league/league.go

```diff
@@ -0,0 +1,35 @@
+package league
+
+import (
+	"encoding/json"
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/utils/log"
+)
+
+func init() {
+	apis.RegisterHandler(fiber.MethodGet, "/leagues", auth.Public, getLeaguesHandler)
+}
+
+func getLeaguesHandler(c *fiber.Ctx) error {
+	db := db.GetSession(c)
+	leagues, err := db.GetLeagues()
+	if err != nil {
+		// todo: create ticket to standardize this error message and pass in model name
+		log.WithErr(err).Alert("Failed to get all leagues from the database")
```

Comment:
I don't believe this works unless you add `log := locals.Logger(c)` to the method

Reply:
I think Ryan is correct. I remember from class on 5/23 Dr. Christensen mentioning locals.Logger(c) will need to be used in every Handler method.